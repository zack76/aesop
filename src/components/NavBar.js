import React, { Component } from 'react';

const NavBar = () => (
    <nav className="navbar navbar-light bg-light text-center">
        <a className="navbar-brand" href="#">Aesop</a>
    </nav>
)

export default NavBar
