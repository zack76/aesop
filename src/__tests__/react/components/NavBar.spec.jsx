import React from 'react';
import { shallow } from 'enzyme';
import NavBar from '../../../components/NavBar';

describe('NavBar', () => {
    it('should render without error', () => {
        const wrapper = shallow(<NavBar debug />);
        expect(wrapper).toMatchSnapshot();
    });

    it('should have a tag and nav tag inside NavBar', () => {
        const wrapper = shallow(<NavBar />);
        expect(wrapper.find('a').length).toBe(1);
        expect(wrapper.find('nav').length).toBe(1);
        expect(wrapper).toMatchSnapshot();
    });
});
