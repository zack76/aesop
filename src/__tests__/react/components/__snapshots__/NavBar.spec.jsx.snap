// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should have a tag and nav tag inside NavBar 1`] = `
<nav
  className="navbar navbar-light bg-light text-center"
>
  <a
    className="navbar-brand"
    href="/#"
  >
    Aesop
  </a>
</nav>
`;

exports[`NavBar should render without error 1`] = `
<nav
  className="navbar navbar-light bg-light text-center"
>
  <a
    className="navbar-brand"
    href="/#"
  >
    Aesop
  </a>
</nav>
`;
